symplyphysics/laws/chemistry/free_path_of_atomic_particles_in_gaseous_medium.py

```diff
@@ -13,6 +13,12 @@
 ## P - pressure,
 ## g - the cross section of the interaction of a traveling atom and a gas atom.
 
+# Links: Wikipedia, the fourth formula <https://en.wikipedia.org/wiki/Mean_free_path#Kinetic_theory_of_gases>
+
+# FIXME: hasn't an additional `sqrt(2)` factor been left out in the denominator?
```

Comment:
Looks like a bug. If possible, find some real numbers to write proper test.

Reply:
Technically this is a "naive" formula that appears in the derivation of the correct formula, so I get where this comes from. Although we should probably only include the correct one.

Reply:
I'll fix this law and test tho.